integration/bitcoind.js

```diff
@@ -152,4 +156,32 @@ describe('Integration with ' + network.name + ' bitcoind', function() {
       peer.sendMessage(message);
     });
   });
+  var from = [blockHash[network.name]];
+  var stop = stopBlock[network.name];
+  it('can get headers', function(cb) {
+    connect(function(peer) {
+      peer.once('headers', function(message) {
+        (message instanceof Messages.Headers).should.equal(true);
+        message.headers.length.should.equal(2);
+        cb();
+      });
+      var message = new Messages.GetHeaders(from, stop);
+      peer.sendMessage(message);
+    });
+  });
+  it.skip('can get blocks', function(cb) {
```

Comment:
`skip` intentional, that test is a WIP. Yes, remocing the `if (message.inventory.length === 2)` fixes it, but the semantics of the test would be wrong. 
Regarding this integration tests, I'm writing them to obtain message dumps, which I then use for unit tests. I figured it would be nice to keep the code, given it may be useful to some as examples.